### ISSUE TITLE
Common Jackson Object Mapper for non-rest use

### ISSUE THREAD

**Issue by terrypacker** [infiniteautomation/ma-core-public#1075]
Since we are using an ObjectMapper in several locations I would suggest that we create a Common one for use in the DB for sure and potentially share this one or create another for use in Data Sources.

See #1069 for where it is currently used.

**Comment by terrypacker** [comment]
I've referenced the commits in issue #1069 but the Dao's now reference a common Object Mapper and is only accessible via the factory methods for performance.

However this is only available to classes that extend BaseDao and Mqtt and NoSQL module need access to an object mapper.  So the question remains do we want to share the same one or create a 3rd mapper for use in data sources/utilities.  

Basically our options are create another one in Common or remove the one from BaseDao and create one in Common that can be used in the Dao methods.

**Comment by terrypacker** [comment]
Changes made to have 3 mappers all customizable from the JacksonModuleDefinition class.

We will need to test these modules due to the changes:

[ ] - Mqtt (MqttClientDataSourceRT)
[ ] - Http (HttpDataSourceDwr, HttpDataSourceServlet, HttpReceiverJsonData, HttpJsonRetrieverDataSourceRT)
[ ] - MangoNoSQL (PointDataTimeSerializer)
